backend/src/apiserver/server/job_server_test.go

```diff
@@ -415,7 +415,7 @@ func TestCreateJob_V2(t *testing.T) {
 
 	// Test all parameters types converted to model.RuntimeConfig.Parameters, which is string type
 	v2RuntimeParams := map[string]*structpb.Value{
-		"param1": {Kind: &structpb.Value_StringValue{StringValue: "world"}},
+		"text":   {Kind: &structpb.Value_StringValue{StringValue: "world"}},
```

Comment:
nit: This change shouldn't be necessary? Looks a bit odd the list is `text`, `param2`, `param3`, ...

Reply:
It is necessary because the pipeline calls for a parameter `text`.

Reply:
I think you are right. I have changed the parameter name to `param1`.